tests/unittests/core/mongodb_test.py

```diff
@@ -18,10 +19,22 @@ def moptdb():
     return moptdb
 
 
+@pytest.fixture()
+def patch_mongo_client(monkeypatch):
+    """Patch ``pymongo.MongoClient`` to force serverSelectionTimeoutMS to 1."""
+    def mock_class(*args, **kwargs):
+        # 1 sec, defaults to 20 secs otherwise
+        kwargs['serverSelectionTimeoutMS'] = 1.
+        return MongoClient(*args, **kwargs)
+
+    monkeypatch.setattr('metaopt.core.io.database.mongodb.MongoClient', mock_class)
```

Comment:
I think you can patch `pymongo.MongoClient` instead. I think that effects are global.